src/main/java/org/opentripplanner/transit/service/TransitModel.java

```diff
@@ -323,20 +343,34 @@ public void addFeedInfo(FeedInfo info) {
    * may become necessary when we start making graphs with long distance train, boat, or air
    * services.
    */
-  public ZoneId getTimeZone() {
-    return timeZone;
+  public Collection<ZoneId> getAgencyTimeZones() {
```

Comment:
You should return a Set here - document unique values, random order. Also, update the JavaDoc.

Reply:
Updated